.github/workflows/create_release.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           SIGNING_STORE_PASSWORD: ${{ secrets.ANDROID_UPLOAD_SIGNING_STORE_PASSWORD }}
           ENCODED_KEYSTORE: ${{ secrets.ANDROID_UPLOAD_KEYSTORE }}
 
-      - name: Upload APK to Github Packages
+      - name: Upload APK to job
```

Comment:
🤔Has the way we create a release changed? I'm not sure whether we need this - maybe we could sync up when you're online? 

I would rather avoid having different workflows (with different steps to be taken) of creating releases if possible. Otherwise things are getting messy really quickly, unless there's a clear documentation what to follow.

Reply:
Uploading the apk to the job results allows us to hand over the result from the build job to the release job. It doesn't have a huge impact on the overall flow.